sources/functions/rtorrent

```diff
@@ -6,23 +6,25 @@ export codename=$(lsb_release -cs)
 
 function whiptail_rtorrent () {
   if [[ -z $rtorrentver ]] && [[ -z $1 ]]; then
+    repoversion=$(apt-cache policy rtorrent | grep focal -B 1 | head -1 | cut -c 6- | cut -c -5)
     if [[ ! ${codename} =~ ("xenial"|"jessie") ]]; then
-      function=$(whiptail --title "Install Software" --menu "Choose an rTorrent version:" --ok-button "Continue" --nocancel 12 50 3 \
+      function=$(whiptail --title "Install Software" --menu "Choose an rTorrent version:" --ok-button "Continue" --nocancel 20 50 5 \
+                  repo "(${repoversion})"\
```

Comment:
The formatting of this should be checked out, I'm not sure I like the way it looks but functionally it does the job
![image](https://user-images.githubusercontent.com/23618693/80156329-27b47700-85b3-11ea-9891-c050a239cb08.png)

Reply:
Decreased height 
![image](https://user-images.githubusercontent.com/23618693/80156978-bbd30e00-85b4-11ea-83c8-5cc7c1883277.png)

Reply:
Code removed from PR for overreach